src/main/java/ai/knowly/langtorch/processor/minimax/embeddings/MiniMaxEmbeddingsProcessorRequestConverter.java

```diff
@@ -0,0 +1,23 @@
+package ai.knowly.langtorch.processor.module.minimax.embeddings;
+
+import ai.knowly.langtorch.llm.minimax.schema.dto.embedding.EmbeddingRequest;
+import java.util.List;
+
+public final class MiniMaxEmbeddingsProcessorRequestConverter {
+  private MiniMaxEmbeddingsProcessorRequestConverter() {}
+
+  public static EmbeddingRequest convert(
+      MiniMaxEmbeddingsProcessorConfig MiniMaxEmbeddingsProcessorConfig,
+      String model,
+      List<String> texts,
+      String type) {
```

Comment:
Does it make sense to make an enum for type?

Reply:
yeah，add class <MiniMaxEmbeddingTypeScene>